docker/postgres-metadb/metadb_s3_tarz

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+#
+# This script runs at metadb PostgreSQL instance as a separate
+# memory-constrained container launched by Airflow scheduler.
+#
+# It creates a base backup in S3 bucket https://ooni-data.s3.amazonaws.com/metadb/
+#
+# It expects AWS_* credentials as environment variables, `awscli` uses
+# AWS_DEFAULT_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY.
+# PG* variables are interpreted by `psql` binary used to call `pg_start_backup()`.
+#
+# It may be replaced by barman (that does not upload to S3 right now) or other solution.
+#
+
+from contextlib import contextmanager
+import concurrent.futures
+import glob
+import gzip
+import logging
+import os
+import queue
+import signal
+import subprocess
+import sys
+import tempfile
+import time
+import urllib.request
+
+assert sys.version_info > (3, 5)
+
+for _ in ('AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'):
+    assert os.environ.get(_), _
+
+assert os.path.isdir('base/pgsql_tmp') and os.access('base/pgsql_tmp', os.W_OK)
```

Comment:
It may be useful to add in the header comment or when the assertion fails that this script is expected to be run with CWD set to the postgres data dir.

Reply:
Yep, Airflow wrapper will do that.